modules/age.nix

```diff
@@ -11,13 +11,15 @@ let
 
   identities = builtins.concatStringsSep " " (map (path: "-i ${path}") cfg.sshKeyPaths);
   installSecret = secretType: ''
-    echo "decrypting ${secretType.file} to ${secretType.path}..."
-    TMP_FILE="${secretType.path}.tmp"
-    mkdir -p $(dirname ${secretType.path})
+    echo "decrypting '${secretType.file}' to '/run/secrets/${secretType.name}'..."
```

Comment:
This is technically a lie, since it's actually decrypted to `/run/secrets.d/[generation]/file`, but I think it's fine.

Reply:
It could just say ‘/run/secrets.d`. I think that’s nicer than lying ;)

Reply:
You're right, fixed :)

Also added a check to the `secretsMountPoint` option to make sure it's non-empty and doesn't have a trailing slash, just like `modules/tasks/filesystems.nix` does.